### PR TITLE
Only run hera on business days and only count business hours

### PR DESCRIPTION
**Pull request by oahayder**
Now that we're enforcing a very granular SLA, best to not count holidays/weekends. Also made the SLA a full 8 hour work day.

```businesstime``` is configurable, but the default work hours are Mon-Fri, 9-5

https://pypi.python.org/pypi/businesstime/0.1.4

### CODE REVIEW

File: bin/issue_reminder.py [hunk 1]
```diff
@@ -9,14 +10,16 @@ class IssueReminder():
     @staticmethod
     def run():
 
+        now = datetime.utcnow()
+        bt = businesstime.BusinessTime(holidays=businesstime.USFederalHolidays())
```
Comment (Garrett-R) [inline bin/issue_reminder.py]: Awesome, even takes account holidays!

File: bin/issue_reminder.py [hunk 2]
```diff
@@ -9,14 +10,16 @@ class IssueReminder():
...
+
         # Dumb. But Heroku scheduler is lame andhas to run everyday
-        d = datetime.now()
-        if d.isoweekday() not in range(1,6):
+        if not bt.isbusinessday(now):
```
Comment (Garrett-R) [inline bin/issue_reminder.py]: Nitpick: Not a big fan of the varname `bt`.  I wouldn't know what that was without checking its initialization.

File: bin/issue_reminder.py [hunk 3]
```diff
@@ -34,9 +37,14 @@ def run():
                             last_updated = pr['updated_at']
                             date = datetime.strptime(last_updated, "%Y-%m-%dT%H:%M:%SZ")
 
-                            delta = datetime.utcnow() - date
+                            # Weird race condition where PRs get updated after we originally assign now which breaks businesstimedelta
```
Comment (Garrett-R) [inline bin/issue_reminder.py]: nitpick: Do you mind wrapping this comment to fit in the column limit?  I can't see the end of it in Github...

File: bin/issue_reminder.py [hunk 4]
```diff
@@ -9,14 +10,16 @@ class IssueReminder():
...
         # Dumb. But Heroku scheduler is lame andhas to run everyday
-        d = datetime.now()
-        if d.isoweekday() not in range(1,6):
+        if not bt.isbusinessday(now):
             return
 
         team_members = {}
         message = ""
-        sla_time_in_hours = 24
+        sla_time_in_hours = 8
```
Comment (Garrett-R) [inline bin/issue_reminder.py]: Why did you decide to put this here and not in, say, `config.py`?

File: bin/issue_reminder.py (hunk 3 above)
Comment (Garrett-R) [inline bin/issue_reminder.py]: Actually, I don't quite get the comment.  It doesn't seem to have a verb in it...

File: bin/issue_reminder.py [hunk 5]
```diff
@@ -34,9 +37,14 @@ def run():
                             last_updated = pr['updated_at']
                             date = datetime.strptime(last_updated, "%Y-%m-%dT%H:%M:%SZ")
 
-                            delta = datetime.utcnow() - date
+                            # Weird race condition where PRs get updated after we originally assign now which breaks businesstimedelta
+                            now = datetime.utcnow()
+                            delta = bt.businesstimedelta(date, now)
```
Comment (Garrett-R) [inline bin/issue_reminder.py]: maybe a better var name for `date` would be `last_updated`, `last_updated_time`, or something?

File: bin/issue_reminder.py [hunk 6]
```diff
@@ -34,9 +37,14 @@ def run():
...
+
                             delta_hours = int(timedelta.total_seconds(delta) / 60 / 60)
 
+                            # If last updated time was over 24hrs of business days, alert
```
Comment (Garrett-R) [inline bin/issue_reminder.py]: Really?  It seems the logic is if the PR hasn't been reviewed for 8 business hours.... or am I missing something?

**Comment by Garrett-R** [comment]
Awesome changes!!

File: bin/issue_reminder.py (hunk 2 above)
Comment (oahayder) [inline bin/issue_reminder.py]: Good point

File: bin/issue_reminder.py (hunk 3 above)
Comment (oahayder) [inline bin/issue_reminder.py]: Lol ya doesn't really make sense. I'll go into a bit more detail.

File: bin/issue_reminder.py (hunk 4 above)
Comment (oahayder) [inline bin/issue_reminder.py]: No reason. Cowboy programming. I'll clean it up.

File: bin/issue_reminder.py (hunk 5 above)
Comment (oahayder) [inline bin/issue_reminder.py]: Yup will do

File: bin/issue_reminder.py (hunk 6 above)
Comment (oahayder) [inline bin/issue_reminder.py]: Nope, I'll update it

File: bin/issue_reminder.py (hunk 4 above)
Comment (Garrett-R) [inline bin/issue_reminder.py]: Haha, sounds good!

**Pull request merged**